source/_includes/compatibility.liquid

```diff
@@ -2,40 +2,40 @@
   <div class="compatibility">
     Compatibility{% if feature %} ({{ feature }}){% endif %}:
   </div>
-  {% if dart != null %}
+  {%- if dart != null -%}
```

Comment:
This fixes whitespace being interepreted as an empty `<p>` and creating extra height in the compatibility block